### DISCUSSION TITLE
gallery: images are download, not there respective links

### DISCUSSION THREAD

**Discussion by t1761** [aclap-dev/video-downloadhelper#59] [Help and Q&A]
### Your question

Since 2 weeks when analyzing a picture page the only visible download option is galery. I'm missing the links option that i could use before that.

**Comment by paulrouget** [comment]
We need more info to help you:

- version number ([see this](https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F)).
- URL of the page

**Reply by t1761** [reply]
Video DownloadHelper
Versie 8.1.6.0
Browsertaal: nl
Productiebuild
Gebouwd op 2024-01-18
https://www.suicidegirls.com/ (pay site)

**Reply by paulrouget** [reply]
Ok - can you try version 8.2: https://downloadhelper.net/beta and tell me if it improves the situations.

If it doesn't, please tell me exactly what the problem is (your first comment is not easy to understand).

**Comment by t1761** [comment]
Hi Paul, 

I tried the beta. The same problem.


A screenprint of a standard photo page in suicidegirls.com

These photos are mine so sharing a view is no issue.
Behind every thumb is a link to a full version.
VDH indicates no media on the page.
When I use the analyse page “gallery of”  shows up. When downloading using this option you get the thumbs.



 

Earlier in addition to galleries I got “links of … “. When downloading the links option I got all the full images to a max of 50. 

 

Hope this clarifies my issue. 

 

Cheers, Ton 

 

 

Van: Paul ***@***.***> 
Verzonden: zaterdag 10 februari 2024 19:45
Aan: aclap-dev/video-downloadhelper ***@***.***>
CC: t1761 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [aclap-dev/video-downloadhelper] links of .... gone (Discussion #59)

 

Ok - can you try version 8.2:  <https://downloadhelper.net/beta> https://downloadhelper.net/beta and tell me if it improves the situations.

If it doesn't, please tell me exactly what the problem is (your first comment is not easy to understand).

**Comment by paulrouget** [comment]
I understand. I'll look into it.

**Comment by paulrouget** [comment]
Dupe of #214
